subprojects/docs/src/docs/release/notes.md

```diff
@@ -169,9 +117,75 @@ In addition, this behavior can be enabled from the command line for individual b
 
 More information is available in the [Java Plugin documentation for test execution](userguide/java_plugin.html#sec:test_execution).
 
-<!--
-### Example new and noteworthy
--->
+### Allow declared reasons for dependency and resolution rules
+
+In complex builds, it can become hard to interpret dependency resolution results and why a [dependency declaration](userguide/customizing_dependency_resolution_behavior.html) or a [rule](userguide/inspecting_dependencies.html#sec:dependency_declaration_reasons) was added to a build script. To improve on this situation, we extended all the corresponding APIs with the capability to define a _reason_ for each declaration or rule. 
+
+These reasons are shown in dependency insight reports and error messages if the corresponding declaration or rule influenced the resolution result. In the future, they will also be shown in build scans.
+
+    dependencies {
+        implementation('org.ow2.asm:asm:6.0') {
+            because 'we require a JDK 9 compatible bytecode generator'
+        }
+    }
+    
+### BOM import
+
+Gradle now [provides support](userguide/managing_transitive_dependencies.html#sec:bom_import) for importing [bill of materials (BOM) files](https://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html#Importing_Dependencies), which are effectively `.pom` files that use `<dependencyManagement>` to control the dependency versions of direct and transitive dependencies. It works by declaring a dependency on a BOM.
```

Comment:
:x: We need to explain which scenarios one can use BOM import to replace the spring dependency management plugin, and when not. How is Gradle's implementation different from Maven here? 

This should be explained in the user manual.

Reply:
I think that would be a useful documentation to have, but its scope goes way beyond the changes here and would impact RC2 release too much.